spec/factories/form526_submissions.rb

```diff
@@ -261,6 +261,36 @@
     end
   end
 
+  trait :with_failed_primary_job do
+    after(:create) do |submission|
+      create(:form526_job_status, :non_retryable_error, form526_submission: submission)
+    end
+  end
+
+  trait :with_exhausted_primary_job do
+    after(:create) do |submission|
+      create(:form526_job_status, :non_retryable_error, form526_submission: submission)
```

Comment:
Should the second argument here be `:exhausted`, instead of `:non_retryable_error`?

Reply:
good catch. it should actually be `pif_in_use_error`, which is a little confusing, but is the existing trait for an 'exhausted' type primary job.